ONSdigital/dp-frontend-renderer
# fix URLs in JSON-LD tags

### What

Updated the logic for building URLs for filterable and legacy dataset pages in the JSON-LD tags. Prior to the fix, the URL for filterable dataset pages would point to the dataset controller rather than the URL on the site

### How to review

- Check that code changes make sense
- Run the dataset controller `develop` branch and the renderer with `ENABLE_JSONLD_CONTROL` set to `true`
- Legacy and filterable dataset pages should now have URLs that match the patterns as specified in the updated unit tests

### Who can review

Anyone but me
